flink-runtime/src/test/java/org/apache/flink/runtime/testutils/TestingJobResultStore.java

```diff
@@ -116,58 +111,53 @@ public static TestingJobResultStore.Builder builder() {
     /** {@code Builder} for instantiating {@code TestingJobResultStore} instances. */
     public static class Builder {
 
-        private ThrowingConsumer<JobResultEntry, ? extends IOException> createDirtyResultConsumer =
-                ignored -> {};
-        private ThrowingConsumer<JobID, ? extends IOException> markResultAsCleanConsumer =
-                ignored -> {};
+        private Function<JobResultEntry, CompletableFuture<Void>> createDirtyResultConsumer =
+                jobResultEntry -> CompletableFuture.completedFuture(null);
```

Comment:
```suggestion
                jobResultEntry -> FutureUtils.completedVoidFuture();
```
That's a really nitty thing. Feel free to ignore it. I just wanted to point out that `FutureUtils` has a utility method for that (that would also apply to some of the other fields if you want to change that).

Reply:
Fixed.